initramfs-init.in

```diff
@@ -209,6 +211,8 @@ configure_ip() {
 		ebegin "Setting IP ($device)..."
 		ip_set "$device" "$client_ip" "$netmask" "$gw_ip"
 		eend $?
+		[ -n "$dns1" ] && echo "nameserver $dns1" >> /etc/resolv.conf
+		[ -n "$dns2" ] && echo "nameserver $dns2" >> /etc/resolv.conf
```

Comment:
Nice, i thought we already had this. If you want you can also move this to a separate commit and ill push it.